src/amo/components/PermissionsCard/index.js

```diff
@@ -0,0 +1,280 @@
+/* @flow */
+import { oneLine } from 'common-tags';
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import log from 'core/logger';
+import { OS_ALL } from 'core/constants';
+import translate from 'core/i18n/translate';
+import { userAgentOSToPlatform } from 'core/installAddon';
+import type { AddonType } from 'core/types/addons';
+import type { ApiStateType, UserAgentInfoType } from 'core/reducers/api';
+import type { I18nType } from 'core/types/i18n';
+import Button from 'ui/components/Button';
+import Card from 'ui/components/Card';
+import Permission from 'ui/components/Permission';
+
+import './styles.scss';
+
+type Props = {|
+  addon: AddonType | null,
+  i18n: I18nType,
+  userAgentInfo: UserAgentInfoType,
+|};
+
+export class PermissionsCardBase extends React.Component<Props> {
+  render() {
+    const { addon, i18n, userAgentInfo } = this.props;
+
+    if (!addon) {
+      return null;
+    }
+
+    // These should be kept in sync with Firefox's strings for webextention permissions
```

Comment:
I discussed this approach at length with @tofumatt and @diox via IRC and we talked about putting something automated in place to pull the strings, in all languages, from Firefox. It was decided that that would be too much work to be worthwhile, and just having our translators translate them, as we do with other strings that also exist in Firefox, would be the most practical approach.

I do have a couple of follow-up issues/ideas though:

1. Even if we won't be automating the process to continually keep our strings in sync with Firefox, perhaps we could run a one-off job to populate all the locales with strings that already exist in Firefox, to save our translators the job of having to either translate or copy all of them.

2. With this approach there is nothing in place to inform us when a new permission and string are added to Firefox, and the way it's coded, a new permission that doesn't have a matching string in this map will simply be ignored, so it seems like it will be very easy to miss one when it is added. What do we think we can do about that? What would be the best way to try to ensure we update this map when new strings are added to Firefox?